config/initializers/rack_attack.rb

```diff
@@ -1,9 +1,7 @@
 LIMIT_LOGIN_ATTEMPTS_PER_IP = 16
 
 Rack::Attack.throttle("limit login attempts per IP", limit: LIMIT_LOGIN_ATTEMPTS_PER_IP, period: 1.hour) do |request|
-  is_welcome_page = request.path == "/" && request.params.dig("user", "email").present?
-  is_login_page = request.path == "/login"
-  if (is_welcome_page || is_login_page) && request.post?
+  if request.path == "/sign-in" && request.params.dig("user", "email").present? && request.post?
```

Comment:
I think this can just be `request.path == "/sign-in" && request.post?`

Reply:
This was causing issues with the JWT tokens being posted in the tests, but that's not an issue now they are only being posted to the welcome controller.